client/src/components/AuthenticationForm.js

```diff
@@ -93,10 +95,35 @@ const authForm = {
 
 function SignInUpform({ authType }) {
```

Comment:
```suggestion
function SignInUpForm({ authType }) {
```

Reply:
The filename and component names are different. Use the same name to make it easier to auto-import your components.